src/phoenix/session/session.py

```diff
@@ -245,17 +246,19 @@ def exports(self) -> ExportedData:
         self.exported_data.add(files)
         return self.exported_data
 
-    def view(self, height: int = 1000) -> "IFrame":
-        """
-        Returns an IFrame that can be displayed in a notebook to view the app.
+    def view(self, *, height: int = 1000, slug: Optional[str] = "") -> "IFrame":
```

Comment:
the code doesn't handle `None` as value
```suggestion
    def view(self, *, height: int = 1000, slug: str = "") -> "IFrame":
```